packages/models/tests/server/organizations_test.coffee

```diff
@@ -23,3 +23,20 @@ describe 'Organization', ->
     organization.set('createdById', 'fakeid')
     organization.save
     expect(organization.createdById).to.eq('fakeid')
+
+  it 'can have members', (test, waitFor) ->
+    userId = 'snoopyId'
+    memberProfile = new UserProfile()
+    memberProfile.set(fullName: 'Snoopy', userId: userId)
+    memberProfile.save()
+    organization.set('name', 'Peanuts')
+    # Checking for errors in the save callback can catch problems with the model.
+    # mUnit only allows one async function per test, so this in not done for
+    # memberProfile.save()
```

Comment:
this is confusing. why can't you do it by defining and wrapping the final callback in waitFor, and then nesting all of them?

something like:
callback = waitFor((err)-> ...)
 memberProfile.save () ->
    organization.save callback